packages/openapi3/test/test-openapi-output.ts

```diff
@@ -857,7 +857,7 @@ async function oapiForModel(name: string, modelDef: string) {
     ${modelDef};
     @route("/")
     namespace root {
-      op read(): ${name};
+      op read(): { @body body: ${name} };
```

Comment:
is there a reason this was changed in the test? Would this be the same?

Reply:
Yes.  This had to be changed because many tests pass the name of a model that is `{}` and expect this to come back as an empty model. We decided that this would translate to no response model, so I could have either changed this or changed all the tests, and changing it here was easier.